internal/tiltfile/config/config.go

```diff
@@ -81,6 +81,19 @@ func (e *Extension) OnStart(env *starkit.Environment) error {
 	}
 
 	err := env.AddValue("config.tilt_subcommand", starlark.String(e.TiltSubcommand))
+	if err != nil {
+		return err
+	}
+
+	err = env.AddValue("config.tiltfile_path", starlark.String(env.StartPath()))
```

Comment:
I think it'd be good to communicate via the name whether this is the root or current tiltfile path. `config.entry_tiltfile_path`? `config.entrypoint_tiltfile_path`?

Reply:
what about entry_path / entry_dir? or maybe main_path / main_dir ?

was trying to find analogues to this in python but was not coming up with good ones.

Reply:
I think python would be `__main__.__file__`. We've already got `__file__`, but not sure we want to go down the road of exposing `__main__` as a thing.

entry_path or main_path both seem fine to me

Reply:
changed it to main_path. ya, agree that __main__ might be a road we eventually go down, but not sure this is the right time.